tests/fv3_conf/hafs_regional_run.IN

```diff
@@ -1,4 +1,10 @@
 rm -rf INPUT RESTART
 mkdir  INPUT RESTART
 
-rsync -arv @[INPUTDATA_ROOT]/FV3_hafs_regional_input_data/. .
+rsync -arv --exclude 'field_table' @[INPUTDATA_ROOT]/FV3_hafs_regional_input_data/. .
+
+# overwrite Field table until the field_table in FV3_hafs_regional_input_data is removed
```

Comment:
Can we remove the if statement here?

Reply:
I think it can, but I'll test it to be certain.